components/climate/index.rst

```diff
@@ -180,6 +180,20 @@ advanced stuff.
       // etc. see API reference
       call.perform();
 
+.. _climate-on_state_trigger:
+
+``climate.on_state`` Trigger
+******************************************************
+
+This trigger is activated each time the climate device is updated (using publish_state).
```

Comment:
`publish_state` is an internal implementation detail that doesn't mean much to our users.
```suggestion
This trigger is activated each time the state of the climate device is updated 
(for example, if the current temperature measurement or the mode set by the users changes).
```

Reply:
Done!